### DISCUSSION TITLE
Hotkeys for highlighting

### DISCUSSION THREAD

**Discussion by blitzuuu** [RyotaUshio/obsidian-pdf-plus#101] [Q&A]
Great plugin, I believe hotkeys for highlighting would make it feel much more fluid. 

Regardless, thanks so much for the plugin!

**Comment by RyotaUshio** [comment]
What do you mean by "hotkeys for highlighting"? There are already the commands "Copy link to selection or annotation"/"Copy link & auto-paste link to selection or annotation", to which you can assign hotkeys. Aren't they what you're looking for?

**Reply by blitzuuu** [reply]
Ah yeah they do work, however I was more thinking hotkeys for the specific colours and such, regardless thanks for the response.

**Reply by samanthavenialogan** [reply]
@RyotaUshio  I am renewing and up-voting this discussion because I agree with @blitzuuu. I would love to assign a key such as "1" to a yellow highlight, but then assign "2" to a blue highlight, so on and so forth for like the first 5 highlights or something like that. that way I can be highlighting with my mouse, and then immediately press the appropriate highlight while in the PDF document. I believe core obsidian would support this but I have no idea how you build the hotkey functions.
